com.unity.visualeffectgraph/Editor/Data/VFXData.cs

```diff
@@ -26,14 +26,22 @@ abstract class VFXData : VFXModel
     {
         public abstract VFXDataType type { get; }
 
-        public virtual uint sourceCount
+        public virtual uint staticSourceCount
         {
             get
             {
                 return 0u;
             }
         }
 
+        public virtual bool hasDynamicSourceCount
+        {
+            get
+            {
+                return false;
+            }
+        }
```

Comment:
Not sure we need virtual interface for this. This looks very tied to particles and multi events

Reply:
Fixed at https://github.com/Unity-Technologies/Graphics/pull/4457/commits/aa8588155096f3380f107f4848c411420ac6ca4e (really minor)